tools/deployment/k8s/s3/deploy.sh

```
@@ -0,0 +1,136 @@
+}
+EOP
+)
+if ! aws s3api put-bucket-policy \
+    --endpoint-url "${AWS_ENDPOINT_URL}" --bucket "${LOG_VIEWER_BUCKET}" --policy "$POLICY"; then
+    log "ERROR" "Failed to set bucket policy for s3://${LOG_VIEWER_BUCKET}"
+    exit 1
+fi
+
+# Download and decompress release tarball to a temp directory
+log "INFO" "Downloading ${RELEASE_TARBALL_URL}"
+readonly DECOMPRESSED_ASSETS_DIRECTORY=$(mktemp -d)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)